doc/RecordingControl.xml

```diff
@@ -1595,6 +1595,89 @@ Change Request 2061, 2063, 2065, 2109</revremark>
         </varlistentry>
       </variablelist>
     </section>
+    <section>
+      <title>ExportRecordedSegments</title>
+      <para>Exports the selected recorded segments (from existing recorded data) to the storage attached to the given recording job.</para>
+      <para>The StoragePath shall be the path as defined in the Object Storage annex appended to the StorageUri from the StorageConfiguration.</para>
+      <variablelist role="op">
+        <varlistentry>
+          <term>request</term>
+          <listitem>
+            <para role="param">Time [tt:DateTimeRange]</para>
+            <para role="param">RecordingToken [tt:RecordingReference]</para>
+            <para role="param">Track [tt:TrackType]</para>
+            <para role="text">An optional field if specified indicates which track is to be exported.</para>
+            <para role="param">OmitSequenceNumber [xs:boolean]</para>
```

Comment:
Purpose of OmitSequenceNumber unclear. If really needed the definition should be positive and default true.

Reply:
It is worded negatively because it overrides what is the default behavior of the cloud upload to specify the "sequence number" of the file. Which now that I review, is actually named `counter` in the Recording Control spec.
So the parameter here would be named `OmitSegmentCounter` probably. And a clearer description.

Reply:
Renamed field to ForceZeroCounter and referenced the annex.